public/src/main/java/com/revenuecat/purchases/models/SubscriptionOption.kt

```diff
@@ -32,5 +32,35 @@ interface SubscriptionOption : Parcelable {
     val isBasePlan: Boolean
         get() = pricingPhases.size == 1
 
+    /**
+     * The recurring [PricingPhase] of the subscription.
+     * Looks for the last finite or infinite recurring price phase of the SubscriptionOption.
+     */
+    val recurringPhase: PricingPhase?
```

Comment:
i'm afraid this might not be specific enough naming... trying to think of a better solution though, it's tough.. 

`terminalPhase`,`fullPricePhase`,`baseSubscriptionPhase`? don't love any of those... but maybe we can keep thinking. i just worry since technically an intro price could be recurring

Reply:
I kind of leaning towards `fullPricePhase` 🤔 

`subscriptionPhase` crossed my mind when doing this but that felt weird and smurfy but it's also accurate?